jax/interpreters/mlir.py

```diff
@@ -986,13 +986,13 @@ def f_lowered(ctx, *args, **params):
 
 def _call_lowering(fn_name, stack_name, call_jaxpr, backend, ctx, avals_in,
                    avals_out, tokens_in, *args):
+  if isinstance(call_jaxpr, core.Jaxpr):
```

Comment:
To confirm, this check exists because not all call primitives use closed jaxprs yet. When they do, we can delete this.